platform/broadcom/sonic-platform-modules-quanta/ix1b-32x/modules/pmbus.h

```diff
@@ -22,109 +22,113 @@
 #ifndef PMBUS_H
 #define PMBUS_H
```

Comment:
is this from linux kernel source tree? if yes, then why not use it directly?

Reply:
Yes it is from kernel source, as u can see other vendors also use this way to include this header file, Is there any other way to achieve this goal?

Reply:
@lguohan
I modified pmbus driver to use it directly. Please help review and merge codes